qa-dcp.planx-pla.net/portal/gitops.json

```diff
@@ -303,10 +303,17 @@
     "guppyConfig": {
       "dataType": "case",
       "fieldMapping": [
-        {
-          "field": "consent_codes",
-          "name": "Data Use Restriction"
-        }
+        { "field": "consent_codes", "name": "Data Use Restriction" },
+        { "field": "project_id", "name": "Project ID" },
```

Comment:
By default Guppy will translate underscore to space and capitalize first letter of each word, so no need to add all those fields, unless if we want to give it a different name.

Reply:
So I guess we could just keep `consent_codes`, and `bmi` (bc need all capitalized) here, and remove the rests